qemu/tcg/tcg.c

```diff
@@ -2375,7 +2375,7 @@ static int tcg_reg_alloc_call(TCGContext *s, const TCGOpDef *def,
     flags = args[nb_oargs + nb_iargs + 1];
 
     nb_regs = ARRAY_SIZE(tcg_target_call_iarg_regs);
-#if TCG_TARGET_REG_BITS == 32
+#if TCG_TARGET_REG_BITS == 32 && defined(_M_IX86)
```

Comment:
@aquynh How about this? If the comment is correct, only microsoft compilers should be affected, so this might be an okay fix?

Reply:
i still dont get it: can you explain why this only affects X86 32bit?

Reply:
That's the commit in question:
https://github.com/unicorn-engine/unicorn/commit/002151874a78c991f41fbe9d16a9f4d6e97ee959
The comment by @xorstream states `// do this because msvc cannot have arrays with 0 entries.`
I didn't look any deeper than that, sorry.